bin/node-scripts/utils.js

```diff
@@ -17,6 +17,13 @@ var sdk = path.join(__dirname, "..", "..");
 var prefsPath = path.join(sdk, "test", "preferences", "test-preferences.js");
 var e10sPrefsPath = path.join(sdk, "test", "preferences", "test-e10s-preferences.js");
 
+var OUTPUT_FILTERS = [
+  /[^\n\r]+WARNING\: NS_ENSURE_SUCCESS\(rv, rv\) failed[^\n]+\n\r?/
+];
+
+var isDebug = (process.env["JPM_FX_DEBUG"] == "1");
+exports.isDebug = isDebug;
```

Comment:
In the future it would be good to have this autodetect based on the firefox binary so I could run this against a custom build more easily